switch_model/wecc/get_inputs.py

```diff
@@ -780,116 +771,76 @@ def connect():
-        )
-        write_csv(
-            "zone_to_regional_fuel_market",
-            ["load_zone", "regional_fuel_market"],
-            db_cursor,
-        )
+    # regional_fuel_markets.csv
+    write_csv_from_query(
+        db_cursor,
+        "regional_fuel_markets",
+        ["regional_fuel_market", "fuel"],
+        f"""
+        select regional_fuel_market, fuel 
+        from switch.regional_fuel_market
+        where regional_fuel_market_scenario_id={regional_fuel_market_scenario_id};
+                    """,
+    )
+
+    # zone_to_regional_fuel_market.csv
+    write_csv_from_query(
+        db_cursor,
+        "zone_to_regional_fuel_market",
+        ["load_zone", "regional_fuel_market"],
+        f"""
+        select load_zone, regional_fuel_market 
+        from switch.zone_to_regional_fuel_market
+        where regional_fuel_market_scenario_id={regional_fuel_market_scenario_id};
+                    """,
+    )
 
     ########################################################
     # DEMAND RESPONSE
     if enable_dr is not None:
-        print("  dr_data.csv...")
-        db_cursor.execute(
-            (
-                """
-			select load_zone_name as load_zone, sampled_timepoint.raw_timepoint_id AS timepoint, 
-			case 
-				when load_zone_id>=10 and load_zone_id<=21 and extract(year from sampled_timepoint.timestamp_utc)=2020 then 0.003*demand_mw
-    			when load_zone_id>=10 and load_zone_id<=21 and extract(year from sampled_timepoint.timestamp_utc)=2030 then 0.02*demand_mw
-    			when load_zone_id>=10 and load_zone_id<=21 and extract(year from sampled_timepoint.timestamp_utc)=2040 then 0.07*demand_mw
-    			when load_zone_id>=10 and load_zone_id<=21 and extract(year from sampled_timepoint.timestamp_utc)=2050 then 0.1*demand_mw
-    			when (load_zone_id<10 or load_zone_id>21) and extract(year from sampled_timepoint.timestamp_utc)=2020 then 0*demand_mw
-    			when (load_zone_id<10 or load_zone_id>21) and extract(year from sampled_timepoint.timestamp_utc)=2030 then 0.03*demand_mw
-    			when (load_zone_id<10 or load_zone_id>21) and extract(year from sampled_timepoint.timestamp_utc)=2040 then 0.02*demand_mw
-    			when (load_zone_id<10 or load_zone_id>21) and extract(year from sampled_timepoint.timestamp_utc)=2050 then 0.07*demand_mw
-    		end as dr_shift_down_limit,
-    		NULL as dr_shift_up_limit
-			from sampled_timepoint
-			left join demand_timeseries on sampled_timepoint.raw_timepoint_id=demand_timeseries.raw_timepoint_id
-			where demand_scenario_id = {id1} 
-			and study_timeframe_id = {id2}
-			order by demand_scenario_id, load_zone_id, sampled_timepoint.raw_timepoint_id;
-						"""
-            ).format(id1=demand_scenario_id, id2=study_timeframe_id)
-        )
-        write_csv(
+        write_csv_from_query(
+            db_cursor,
             "dr_data",
             ["LOAD_ZONE", "timepoint", "dr_shift_down_limit", "dr_shift_up_limit"],
-            db_cursor,
+            f"""
```

Comment:
I noticed the query we have for DR could be improved by creating a table for DR scenarios and pulling data from there. This could be added to a to-do list for the futute.